pages/common/ern.md

```diff
@@ -0,0 +1,36 @@
+# ern
+
+> Electrode Native platform command line client.
+> More information: <https://native.electrode.io/reference/index-6>.
+
+- Create a new `ern` application (`MiniApp`):
+
+`ern create-miniapp {{appName}}`
+
+- Run one or more `MiniApps` in the iOS Runner application:
+
+`ern run-ios`
+
+- Run one or more `MiniApps` in the Android Runner application:
+
+`ern run-android`
+
+- Create an Electrode Native container:
+
+`ern create-container --miniapps {{/absolute/path/to/miniapp_directory}} --platform {{ios|android}}`
+
+- Publish an Electrode Native container to local Maven repository:
+
+`ern publish-container --publisher {{maven}} --platform {{android}} --extra {{'{"groupId":"com.walmart.ern","artifactId":"quickstart"}'}}`
+
+- Transform an iOS container to a pre-compiled binary framework:
+
+`ern transform-container --platform {{ios}} --transformer {{xcframework}}`
+
+- List all installed versions of Electrode Native:
+
+`ern platform versions`
+
+- Set the `ern` tool logging level:
```

Comment:
```suggestion
- Set a specific logging level:
```

This page is all about `ern` tool so it's not necessary mention `ern` explicitly in examples.

Reply:
Another great point, fixed.